spec/factories/user.rb

```diff
@@ -5,6 +5,8 @@
     display_name "jgu"
     password "gu"
     role "default"
+    provider "twitter"
+    uid "abc123"
```

Comment:
Ehh, these are kind of strange factories. Its not a problem (more of a nitpick), but we should try and keep their fields to something that will be universally understood. Perhaps something like:

```ruby
     email "default.user@gmail.com"
     full_name  "Default User"
     display_name "default_user"
     password "password"
     role "default"
```

Just a thought. It's not a high priority, but it's a quick fix.